ladder/teams/README.md

```diff
@@ -0,0 +1,64 @@
+# How Cilium GitHub Teams Are Managed
+The Cilium project makes use of GitHub teams to help enforce responsibility boundaries with regard to the project codebase here on GitHub.
+GitHub teams loosely map groups of [contributors](https://github.com/cilium/community/blob/main/CONTRIBUTOR-LADDER.md) to the Cilium Project as either reviewers and/or committers, focused on a portion of the codebase. [GitHub teams](https://github.com/orgs/cilium/teams) used to manage code access and review responsibilities.
+
+For example, the Cilium Project Code team, may require multiple GitHub teams in the Cilium GitHub organization to self-organize their work. 
+These multiple GitHub teams will be composed of contributors at different steps along the contributor ladder: organizational members, reviewers, and committers.  
```

Comment:
One more - GitHub teams are typically given write privileges and therefore org members are generally not placed on individual teams. In the strict sense, there is a GitHub team in the organization called "Organization Members", and this correlates to the list of names in the "members.yaml" file. But all other teams defined in this repo are for reviewers of specific areas of the Cilium project, mostly under cilium/cilium (as defined by the CODEOWNERS file in that repo).

Reply:
I guess the text is accurate but could be a bit misleading. That is, I would strongly consider whether an org member could join a specific team because for each file under `ladder/teams`, the members listed are expected to be reviewers right now. Therefore adding someone to one of those teams is actually promoting them up the ladder, with corresponding expectations and privileges. We can probably just leave this as-is since the most likely readers of this are reviewers anyway though.